inst/qml/ConfirmatoryFactorAnalysis.qml

```diff
@@ -186,29 +186,29 @@ Form
 		title: qsTr("Advanced")
 		RadioButtonGroup
 		{
-			title: qsTr("Emulation")
-			name: "mimic"
-			RadioButton { label: qsTr("None")  ; value: "lavaan"  ; checked: true }
+			title: qsTr("Package")
```

Comment:
so regardless of what the uses chooses, the underlying package is always `lavaan`, right? So `emulation` does seem more appropriate (as we don't actually use Mplus or EQS to compute the results)...

Reply:
Okay. I just couldn't really make sense of the word emulation... why not use `Mimic package`

Reply:
so maybe title = `Mimic package` and name = `packageMimiced`

Reply:
yeah, that would work :)